go.mod

```diff
@@ -18,6 +19,7 @@ require (
 	k8s.io/klog v1.0.0
 	k8s.io/kube-openapi v0.0.0-20200121204235-bf4fb3bd569c // indirect
 	sigs.k8s.io/controller-runtime v0.5.2
+	sigs.k8s.io/yaml v1.1.0 // indirect
```

Comment:
can you remove this as it is not used?

Reply:
this actually is needed, i am using the yaml package in templates.go